libdeno/binding.cc

```diff
@@ -322,6 +396,43 @@ bool Execute(v8::Local<v8::Context> context, const char* js_filename,
   return ExecuteV8StringSource(context, js_filename, source);
 }
 
+v8::Local<v8::Object> InitializeConstants(v8::Isolate* isolate,
+                                          v8::Local<v8::Context> context) {
+  auto constants_val = v8::Object::New(isolate);
+
+  auto promise_reject_event_val = v8::Object::New(isolate);
+  CHECK(promise_reject_event_val
+            ->Set(context, deno::v8_str("kPromiseRejectWithNoHandler"),
```

Comment:
put this directly into the libdeno namespace, not `libdeno.constants.promiseRejectEvents`